### PR TITLE
Fix headings in accordance with WCAG

### PR DESCRIPTION
**Pull request by podliashanyk**
Following WCAG requirements are satisfied:

- WCAG 2.0 - 1.3.1
- WCAG 2.0 - 2.4.10
- WCAG 2.0 - 2.4.6

In other words:

- One h1 per page (hidden in our case)
- All headings are in logical order and there are no "jumps" between headings (f.e. from h2 to h4)
- HTML headers are not used as a replacement for lists
- All sections have headers
- All headers have descriptive and informative names
- Everything that looks like heading is coded as one

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -25,6 +25,9 @@
         </div>
         <div id="features" class="container content">
             <div class="row">
+                <div class="col-md-12">
+                    <h2 class="super-heading">Features</h2>
```
Comment (lunkwill42) [inline layouts/index.html]: This actually kind of breaks the design a bit, IMHO. For the seeing, it is obvious that this section lists features, with the four large icons describing the main features.  I suggest this heading is also hidden.

**Review by lunkwill42 — CHANGES_REQUESTED** [review]
This is nice, but I don't like the new "Features" heading as is.

Also, since you're based on the `migrate-blog` branch in this instance, there is a much-needed followup to this in the blog list page: The markdown renderer that renders the content of each individual blog posting in the list starts all headings at `<h1>`, causing multiple of them in the same page. We need to find out whether there is some way to tell the renderer to start at `<h2>` instead.

**Comment by podliashanyk** [comment]
> Also, since you're based on the `migrate-blog` branch in this instance, there is a much-needed followup to this in the blog list page: The markdown renderer that renders the content of each individual blog posting in the list starts all headings at `<h1>`, causing multiple of them in the same page. We need to find out whether there is some way to tell the renderer to start at `<h2>` instead.

Since one and just one h1 per page is not enforced by Norwegian law (it is a AAA requirement), and since I haven't found a quick fix for this, I have created #13 to work on this issue separately. 🙂

**Review by lunkwill42 — APPROVED** [review]
:+1: But there's a merge conflict now that you have so many PR's in-flight ...

**Pull request merged**